### PR TITLE
Get better Lighthouse score in a11y and performance

### PR DESCRIPTION
**Pull request by myfrom**
I did a few things (all mentioned in the commits) to improve the Lighthouse score. There are a few differences between my tests and the test file from https://gci-leaders.netlify.com/ that were caused by my development environment and the fact that machine is fairly slow.

I also changed the output of /static from /out/static to the root of /out as we were discussing in #55

Here are test results before and after:
[tests.zip](https://github.com/coala/gci-leaders/files/1552817/tests.zip)

Related to #55, part of [GCI task](https://codein.withgoogle.com/dashboard/task-instances/5311708543320064/)

### CODE REVIEW

**Review by andrewda — COMMENTED** [review]
Not sure why Lighthouse didn't complain about this, but the RocketChat logo can probably be shrunk to 72px.

File: templates/main.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8">
     <title>GCI Leaders</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=5.0, user-scalable=yes">
```
Comment (blazeu) [inline templates/main.html]: I think `maximum-scale=5.0, user-scalable=yes` is not needed now.

**Review by blazeu — CHANGES_REQUESTED** [review]
Favicon has white border, looks ugly, remove them.

Also, I don't know if this is accurate or not, but checked via https://realfavicongenerator.net/favicon_checker
image

favicon.ico should contain multiple sizes (in the same file).

**Review by blazeu — CHANGES_REQUESTED** [review]
Commit message should be in imperative present tense https://coala.io/commit
e.g. Improved ~> Improve

**Comment by jayvdb** [comment]
@myfrom , please undo your merge commit.  We do proper rebases at coala.  see http://coala.io/rebase .

Also please in future do not create pull requests from your 'master' branch. (but please do not close this PR; we'll get this one fixed & merged ; ask us for help on Zulip)

**Comment by myfrom** [comment]
@andrewda @jayvdb @blazeu Fixed everything

**Comment by blazeu** [comment]
If you turn the GCI logo on the website into png or vector with transparent background, it might reduce the size. Try it first.

**Comment by blazeu** [comment]
Also, your branch is still outdated, you need to rebase. http://coala.io/rebase

**Comment by myfrom** [comment]
Oh, sorry, should be OK now.

**Review by blazeu — CHANGES_REQUESTED** [review]
Some of the images are broken. See https://deploy-preview-78--gci-leaders.netlify.com/ (deploy/netlify on below)

And wiki icons are gone, you need to add them back, resize if needed, and add the `alt`

**Comment by myfrom** [comment]
Oh, sorry, missed those new icons. I updated them and basically have everything working, I'll upload tomorrow.

@blazeu
But I have one big problem: someone made a script that instead of creating new data.json takes the current one. The current one is pointing to images in static which now are in root. How should I overcome this? Delete this script?

**Comment by andrewda** [comment]
We should add a CACHE_VERSION and increment it whenever anything breaking occurs to the data. This will let us bust the cache if it becomes stale. I can do this in a few hours.

**Comment by blazeu** [comment]
Why not just remove the image url from the JSON and then rely on the chat platform name.

Edit: @myfrom ^ You should do that

**Comment by andrewda** [comment]
That would be one option, but adding a CACHE_VERSION would allow us to avoid this problem entirely in the future, such as if a new key were added or an existing one were taken away.

**Comment by blazeu** [comment]
In the mean time, I'm gonna run `npm run build:clean` to ignore cache on netlify so that @myfrom can continue his work.

**Comment by myfrom** [comment]
Should be OK now if we run it through `$ npm run build:clean`

**Comment by blazeu** [comment]
Okay.

After this one, I want you sent another PR to `gh-pages` to ~~delete~~ update `data.(js|min.js|yml)` to remove those old keys. (or anybody else or me).

Edit: ^ Done.

**Review by blazeu — APPROVED** [review]

**Comment by blazeu** [comment]
ack https://github.com/coala/gci-leaders/pull/78/commits/be803a965a8cef653d8231c0e66843fce0dad980 https://github.com/coala/gci-leaders/pull/78/commits/f95ffaf214f529b69de444760653ce75f2621410 https://github.com/coala/gci-leaders/pull/78/commits/cf4a35d4e13be80a773d0b316be639e1bb6780a7 https://github.com/coala/gci-leaders/pull/78/commits/206a75c24f14f10f744a574c87920d6d07dfe218

**Pull request merged**